### PR TITLE
Add song minsan by eraserheads

### PR DESCRIPTION
**Pull request by fafansin**
Please review this @casey-collab

### CODE REVIEW

File: Philippines/kanta/minsan.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minsan sa may kalayaan tayo'y nagkatagpuan
```
Comment (casey-collab) [inline Philippines/kanta/minsan.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Philippines/kanta/minsan.md (hunk 1 above)
Comment (fafansin) [inline Philippines/kanta/minsan.md]: Added the link from samalat.md @casey-collab

**Comment by fafansin** [comment]
Added the link from salamat.md @casey-collab

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**